jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -270,3 +328,32 @@ def _get_partners_by_batch_type(cls, batch_type) -> List[CorpTypeModel]:
             filter(DistributionCodeLinkModel.distribution_code_id.in_(fee_distribution_codes)).all()
 
         return db.session.query(CorpTypeModel).filter(CorpTypeModel.code.in_(corp_type_codes)).all()
+
+    @classmethod
+    def _process_partial_refunds(cls, batch_type, refund_partial_items, ejv_content, control_total,  # pylint: disable=too-many-arguments
+                                 partial_refund_number, credit_distribution, debit_distribution,
+                                 journal_name, effective_date):
+        for refund_partial in refund_partial_items:
+            # JV Details for partial refunds
+            partial_refund_number += 1
+            flow_through = (
+                f'Partial Refund#{refund_partial.id} '
+                f'for Payment Line Item#{refund_partial.payment_line_item_id:<110}'
+            )
+            description = f'Partial Refund {refund_partial.id} for PLI {refund_partial.payment_line_item_id}'
+            description = f'{description[:100]:<100}'
```

Comment:
Probably need something like this for flow_through, as discussed this should be a number and some way of differentiating between invoices and refunds

Reply:
Take a look at the feedback files for reference

Reply:
updated, changed flowthrough with refunds_partial id